docs/platform/howto/support.md

```diff
@@ -121,36 +102,34 @@ To add Aiven users to a support ticket:
 1.  On the **My tickets** page, open the ticket.
 1.  Click **Add to conversation**.
 1.  Add the email addresses in the **CC** field separated by a space.
-    This must be the same email address they use to log in.
+    These must be the same email addresses they use to log in.
 1.  Enter a comment and click **Submit**.
 
 ## Get notifications for all support tickets
 
-Super admin can get notifications for updates on all tickets in their
-organization.
+[Super admin](/docs/platform/howto/make-super-admin) can get notifications for
+updates on all tickets in their organization.
 
 1.  Click **My tickets**.
-1.  On the **Tickets in my organization** tab, click **Follow all
-    tickets**.
+1.  On the **Tickets in my organization** tab, click **Follow all tickets**.
 
 You will get email notifications for all updates on both existing and
 new tickets. You can unfollow them at any time.
```

Comment:
@staceysalamon-aiven one thing would be nice to add: if you want to allow all user in your org (not only admins) to follow on all tickets (just like admins) - contact your account team and they can make it happen.

Reply:
It's already there: https://github.com/aiven/aiven-docs/blob/c9e890d896ccd16805a256219064f88d4cf4355f/docs/platform/howto/support.md?plain=1#L97-L98